src/sentry/static/sentry/app/views/eventsV2/data.tsx

```diff
@@ -30,7 +30,7 @@ export const AGGREGATE_ALIASES = [
   'p99',
   'last_seen',
   'latest_event',
-] as const;
+];
```

Comment:
Forcing this to const made using this value really annoying as you couldn't do `.includes()` on it with a string as typescript would fail.